tests/radar_client.unit.test.js

```diff
@@ -472,58 +487,72 @@ exports.RadarClient = {
         done();
       },
 
       'duplicated subscribes and syncs should only be stored once and sync is more important than subscribe': function(done) {
         // Simple duplicates
-        client._memorize({ op: 'subscribe', to: 'foo'});
+        var request = Request.buildSubscribe('status:/test/ticket/1');
+        client._memorize(request);
         assert.equal(1, Object.keys(client._subscriptions).length);
-        client._memorize({ op: 'subscribe', to: 'foo'});
+        client._memorize(request);
         assert.equal(1, Object.keys(client._subscriptions).length);
 
 
         client._subscriptions = {};
         // Simple duplicates
-        client._memorize({ op: 'sync', to: 'abc'});
+        request = Request.buildSync('status:/test/ticket/2');
+        client._memorize(request);
         assert.equal(1, Object.keys(client._subscriptions).length);
-        client._memorize({ op: 'sync', to: 'abc'});
+        client._memorize(request);
         assert.equal(1, Object.keys(client._subscriptions).length);
 
         client._subscriptions = {};
         // Sync after subscribe
-        client._memorize({ op: 'sync', to: 'bar'});
+        request = Request.buildSubscribe('status:/test/ticket/3');
+        client._memorize(request);
         assert.equal(1, Object.keys(client._subscriptions).length);
-        client._memorize({ op: 'sync', to: 'bar'});
+        request = Request.buildSync('status:/test/ticket/3');
+        client._memorize(request);
         assert.equal(1, Object.keys(client._subscriptions).length);
-        assert.equal('sync', client._subscriptions.bar);
+        assert.equal('sync', client._subscriptions['status:/test/ticket/3']);
 
         client._subscriptions = {};
         // Subscribe after sync
-        client._memorize({ op: 'sync', to: 'baz'});
+        request = Request.buildSync('status:/test/ticket/4');
+        client._memorize(request);
         assert.equal(1, Object.keys(client._subscriptions).length);
-        assert.equal('sync', client._subscriptions.baz);
+        assert.equal('sync', client._subscriptions['status:/test/ticket/4']);
         // When we sync and subscribe, it means just sync
-        client._memorize({ op: 'subscribe', to: 'baz'});
+        request = Request.buildSubscribe('status:/test/ticket/4');
+        client._memorize(request);
         assert.equal(1, Object.keys(client._subscriptions).length);
-        assert.equal('sync', client._subscriptions.baz);
+        assert.equal('sync', client._subscriptions['status:/test/ticket/4']);
 
         done();
       }
     },
+
     '_restore' : {
       'restore presences' : function(done){
         MockEngine.current._written = [];
-        client._memorize({ op: 'set', to: 'presence:/foo/bar', value: 'online' });
-        client._memorize({ op: 'set', to: 'presence:/foo/bar2', value: 'offline' });
+        var request = Request.buildSet('presence:/foo/bar', 'online');
```

Comment:
I think the *isEqualTo* might need to be different in different cases, so I don't plan to do this right now.  In many of the existing tests, I modified as little as was required.  We can review and refactor tests later.